manifests/0011_configmap_special-resource-depedencies.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+data: null
+kind: ConfigMap
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: special-resource-depedencies
```

```suggestion
  name: special-resource-dependencies
```
and the same for the filename

---

This is autogenerated fixed in `config`